ui/00_message/skill/custom_skill_info_03.xml

```diff
@@ -85,7 +85,7 @@ LV.6: Increases the range of the effect.&lt;/COL&gt;</translation>
 LV.６で溜め段階拡張（１→２）&lt;/COL&gt;</original>
     <translation>Draws the bow with all the user's might,
 then looses a shot straight ahead.
-Power increases if released immediately.
+Power greatly increases if released as charge reaches max.
```

Comment:
Feel like this would be better worded as "Power greatly increases if released as maximum charge is reached."?